src/main/scala/org/squbs/sample/SampleHttpService.scala

```diff
@@ -63,7 +63,7 @@ class SampleHttpService extends RouteDefinition {
   private val chunks =
     path("hello" / Segment / IntNumber) { (who, delay) =>
       onComplete(ActorLookup ? ChunkRequest(who, delay milliseconds)) {
-        case Success(source: Source[PingResponse, Any]) =>
+        case Success(source: Source[_, _]) =>
```

Comment:
> Source `types` are not strictly necessary for this pattern match.

It actually is in line 70, when we do `source.map`, this would still give a compiler warning for the case match at line 71.